src/main/features/response/form/models/statement-of-means/dependants.ts

```diff
@@ -17,28 +16,13 @@ export class Dependants implements Serializable<Dependants> {
   hasAnyChildren: boolean
 
   @ValidateIf(o => o.hasAnyChildren === true)
-  @IsDefined({ message: ValidationErrors.UNDER_11_REQUIRED })
-  @IsInt({ message: GlobalValidationErrors.NUMBER_REQUIRED })
-  @Min(0, { message: GlobalValidationErrors.NUMBER_REQUIRED })
-  under11: number
+  @ValidateNested({ each: true })
```

Comment:
I think we do not need to say `{ each: true }` This is just for validating elements of collection while here we have just one instance.

Reply:
ok

Reply:
remove attr